src/huntsman/pocs/observatory.py

```diff
@@ -476,6 +476,12 @@ def prepare_cameras(self, sleep=60, max_attempts=5, require_all_cameras=False):
                                   f'waiting another {sleep} seconds before checking again.')
                 time.sleep(sleep)
 
+        # Remove cameras that didn't become ready in time
+        # This must be done outside of the main loop to avoid a RuntimeError
+        for cam_name in cameras_to_drop:
+            self.logger.debug(f'Removing {cam_name} from {self} for not being ready.')
```

Comment:
Just a question: what does this print in 'self'? is that the list of cameras or the class?

Reply:
Hi @JAAlvarado-Montes , thanks for looking at this! In this case the function is a method of the `HuntsmanObservatory` class, so `self` will refer to a `HuntsmanObservatory` instance. 

You might find this helpful:  https://stackoverflow.com/questions/1535327/how-to-print-instances-of-a-class-using-print